packages/search/src/util/aggregations/merge-aggregations.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * Interface for an Aggregation returned from search request,
+ * where label is a value for a fieldname type for which aggregations were requested
+ * and aggValue is the aggregation value
+ */
+interface IAggregation {
+  label: string;
+  aggValue: number;
+}
+
+/**
+ * Interface for an list of Aggregations requested for a particular fieldname
+ * Ex. { fieldName: 'access', aggregations: [ { public: 2, private: 5, ... } ] }
+ */
+export interface IAggregationResult {
+  fieldName: string;
+  aggregations: IAggregation[];
+}
+
+/**
+ * A function used when to merge the same aggregation across multiple aggregation sources.
+ * Implementations can handle summation of two aggregation values (as with COUNT and SUM),
+ * but can also handle MIN/MAX
+ */
+type mergeFunc = (aggValueOne: number, aggValueTwo: number) => number;
+
+/**
+ * Alias types helpful for readability when processing list of aggregations results
+ */
+type AggregationMap = Record<string, number>;
+type AggregationResultMap = Record<string, AggregationMap>;
+
+/**
+ * The default merge function for merging aggregations. Simply sums the aggregations
+ * @param aggValueOne the value of one aggregatiom
+ * @param aggValueTwo the value of the second aggregation
+ * @returns the sum of the two values
+ */
+const defaultMergeFunc = (aggValueOne: number, aggValueTwo: number) => {
```

Comment:
suggest calling this `sumAggregations`

Reply:
Good call. It's use as a default argument makes `defaultMergeFunc` redundant as a name.